gammapy/datasets/flux_points.py

```diff
@@ -55,6 +58,19 @@ class FluxPointsDataset(Dataset):
     meta_table : `~astropy.table.Table`
         Table listing information on observations used to create the dataset.
         One line per observation for stacked datasets.
+    stat_type : bool
+        Method used to compute the statistics:
+        * chi2 : etimate from chi2 statistics.
+        * profile : estimate from interpolation of the likelihood profile.
+        * distrib : estimate from probability distributions
+                    assuming that flux points correspond to asymmetric gaussians
+                    and upper limits complemantary error functions.
+        Default is `chi2`, in that case upper limits are ignored and the mean of asymetrics error is used.
+        However it is recommended to use `profile` if `stat_scan` is available on flux points.
+    stat_kwargs : int
+        Exta arguments passed to the stat function
```

Comment:
It is difficult to understand what can be configured. Maybe 

Extra arguments specifying the interpolation scheme of the likelihood profile. Used only in case `stat_type=="profile"`